python/hail/tests.py

```diff
@@ -197,8 +197,8 @@ def test_select(self):
         self.assertEqual(set(kt.select(kt.a, foo=kt.a + kt.b - kt.c - kt.d, **kt.g).columns), {'a', 'foo', 'x', 'y'})
 
     def test_aggregate(self):
-        schema = hl.tstruct(['status', 'GT', 'qPheno'],
-                            [hl.tint32, hl.tcall, hl.tint32])
+        schema = hl.tstruct.from_lists(['status', 'GT', 'qPheno'],
```

Comment:
we use `quant_pheno` elsewhere, might as well be consistent / pythonic

Reply:
orthogonal.